before_install_linux.sh

```diff
@@ -15,35 +15,50 @@ if [ ! -d /usr/include/freetype ]; then
   sudo ln -sf /usr/include/freetype2 /usr/include/freetype
 fi
 
+dir_options=""
+if [ -v CONFIGURE_OPTIONS ]; then
```

Comment:
I don't think we need to add any other configure options so it might be better to just rename `CONFIGURE_OPTIONS` into `ENABLE_HDRI` and then use the to create the directory based on that. If you need some help with that please let me know 👍

Reply:
I can follow up with this after this PR is merged since the diff here is already pretty big.